torch/fx/graph.py

```diff
@@ -539,6 +539,9 @@ def emit_node(node : Node):
                 # use string as annotation, to make it valid python code
                 if isinstance(meta_val, FakeTensor):
                     maybe_type_annotation = f': "{dtype_abbrs[meta_val.dtype]}{stringify_shape(meta_val.shape)}"'
+                    sparsity = meta_val.sparsity
+                    if sparsity is not None:
```

Comment:
What is the value of `meta_val.layout` when `sparsity` is not `None`?

Reply:
see discussion elsewhere, we may be able to do without sparsity once your RFC of having compressed "meta" is done